### DISCUSSION TITLE
Google Cloud Monitoring: Make deep linking to metrics explorer optional

### DISCUSSION THREAD

**Discussion by merlinran** [grafana/grafana#39816] [Data Sources]
It's nice to [have a data link to Metrics Explorer](https://grafana.com/docs/grafana/latest/datasources/google-cloud-monitoring/#deep-linking-from-grafana-panels-to-the-metrics-explorer-in-google-cloud-console), but that's not always desirable. In my case, I have Stat panels in the main dashboard and each have drill down links to a detailed dashboard. When the Stat panels use Cloud Monitoring data source, the embedded data link always overrides the links I configured.

I think there can be two improvements here:
1. Allow user to disable the automatic deep linking.
2. By default, show the the deep link to Metrics Explorer along with user configured data links (if any).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
